underscore.js

```diff
@@ -1370,6 +1370,24 @@
 
   _.property = property;
 
+  // Generates a function for a given object that returns the passed function's return value.
+  // Accepts a method name or function literal for value.
+  // If value is a string, function assumes this string is a method name on the
+  // referenced object.
+  _.method = function(value) {
+    var isFunctionLiteral = _.isFunction(value),
+        func;
+
+    return restArgs(function(obj, args) {
+      if (obj == null) { return; }
```

Comment:
My current rational is that this invocation pattern doesn't make much sense to me.  e.g.,

```
_().method(function() { /* function body *. })
_.method(null, function () { /* function body */ })
```

since both would be more easily written as a plain function literal, and this invocation pattern reads to me as 'call the provided method on undefined'.

That said, if allowing an undefined object in this position is more in keeping with underscore's style, I'm happy to update so we can call _.method on undefined objects.